### PR TITLE
updated ocp cli version

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #1307

### CODE REVIEW

**Comment by dgolovin** [comment]
@sudhirverma have you had a chance to try it with 3.x?

**Comment by sudhirverma** [comment]
@dgolovin I have check on cluster 3.x it was working fine.

File: src/tools.json [hunk 1]
```diff
@@ -33,28 +33,28 @@
         "vendor": "Red Hat, Inc.",
         "name": "oc",
         "cmdFileName": "oc",
-        "version": "3.11.0",
-        "versionRange": "^3.11.0",
-        "versionRangeLabel": "version >= 3.11.0 and < 4.0.0",
+        "version": "4.2.2",
+        "versionRange": "^4.2.2",
+        "versionRangeLabel": "version >= 3.11.0 and <= 4.2.2",
```
Comment (dgolovin) [inline src/tools.json]: ^4.2.2 version range means `version >= 3.11.0 and < 5.0.0` according to semver module doc https://www.npmjs.com/package/semver#caret-ranges-123-025-004.
We should allow to use oc found in path if it differs only in patch version ~4.2.2 which means `version >= 4.2.2  and < 4.3.0`.

File: src/tools.ts [hunk 2]
```diff
@@ -106,7 +106,12 @@ export class ToolsConfig {
     public static async getVersion(location: string, cmd: string = path.parse(location).name): Promise<string> {
         let detectedVersion: string;
         if (fs.existsSync(location)) {
-            const version = new RegExp(`${cmd.toLocaleLowerCase()} v((([0-9]+)\\.([0-9]+)\\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?)(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?).*`);
+            let version: RegExp;
```
Comment (dgolovin) [inline src/tools.ts]: This can be single RegExp expression.

**Comment by dgolovin** [comment]
Replaced with #1387.

**Pull request closed without merging**